components/common-go/baseserver/server.go

```diff
@@ -298,6 +298,7 @@ func (s *Server) initializeGRPC() error {
 		opts = append(opts, grpc.Creds(credentials.NewTLS(tlsConfig)))
 	}
 
+	opts = append(opts, grpc.MaxRecvMsgSize(100*1024*1024))
```

Comment:
Can we not change the gRPC base server default message size and let the caller overwrite the default message size? Because this is an optional for all gRPC server, WDYT?

Reply:
Yes, we can probably provide an option to set this. 

The question is whether a user of the baseserver would ever care that the maximum message size was too large. IMO, probably not in which case an option is not needed. Perhaps we should add the option when it's needed.